lib/Reports/SingleUser.php

```diff
@@ -149,11 +143,7 @@ protected function getUserLastLogin(string $userId): int {
 		return (int)$lastLogin;
 	}
 
-	/**
-	 * @param string $userId
-	 * @return int|string
-	 */
-	protected function getUserQuota(string $userId) {
+	protected function getUserQuota(string $userId): int|string {
```

Comment:
Not sure if this apps supports 32bits, but if it does it should use `int|float` for quotas.